morpheus/messages/multi_inference_message.py

```diff
@@ -220,9 +205,12 @@ def get_input(self, name: str):
         cupy.ndarray
             Inference input.
 
+        Raises
+        ------
+        AttributeError
+            When no matching input tensor exists.
         """
-
-        return self.memory.tensors[name][self.offset:self.offset + self.count, :]
+        return self.memory.get_input(name)[self.offset:self.offset + self.count, :]
```

Comment:
Seeing this leads me to believe that we need one more layer of abstraction: `TensorMultiMessage` which is the base for `InferenceMultiMessage` and `ResponseMultiMessage`. `TensorMultiMessage` should handle calculating the offsets like this and the derived classes just call `get_tensor`/`set_tensor` with specific names.